var/spack/repos/builtin/packages/ompss-2/package.py

```diff
@@ -31,6 +31,8 @@ class Ompss2(Package):
     depends_on("python", type="build")
     depends_on("cmake", type="build")
     depends_on("extrae", when="+extrae")
+    depends_on("boost@1.59.0:")
+    depends_on("numactl")
```

Comment:
Do these apply for a specific version on? Or for all listed versions?

Reply:
They apply to all versions